README.md

```diff
@@ -49,6 +49,7 @@ _Libraries for use in ASGI apps._
 - [HTTPX](https://www.encode.io/httpx) - Next generation HTTP client, including async support and ability to call ASGI apps directly.
 - [Mangum](https://github.com/erm/mangum) - AWS Lambda & API Gateway support for ASGI.
 - [python-socketio](https://python-socketio.readthedocs.io) - WebSocket clients and servers using Socket.IO. Includes an ASGI application wrapper.
+- [Scout-APM](https://pypi.org/project/scout-apm/) - Application Performance Monitoring agent supporting Django, Starlette, and Starlette-based frameworks.
```

Comment:
It seems the integration is done via an ASGI middleware. Could we move this to the middleware section, and possibly update the link to point at docs specifically about the ASGI integration?

Also, not too keen on mentioning the Django integration here, since that’s not specifically related to ASGI. (You can probably submit this to a Django awesome list, though. :-))

So, an example rewording suggestion:

```markdown
- [Scout-APM Starlette](https://docs.scoutapm.com/#starlette) An ASGI middleware for integrating Scout-APM with Starlette and Starlette-based frameworks. (Part of the `scout-apm` package.)
```

Reply:
Sure. It's technically more than a middleware although from the user's perspective only the middleware needs setting up. I think for ASGI a lot of things will work like that since it's "turtles all the way down".. not sure of the value long term of splitting "middleware" from "libraries."

Have moved and adapted your wording to be closer to Sentry which is in a similar situation.

Reply:
> not sure of the value long term of splitting "middleware" from "libraries."

Yes, I think it's a valid point. :-) Opened https://github.com/florimondmanca/awesome-asgi/issues/25 to discuss a possible review of the current categories.